src/cljdoc/render/api.clj

```diff
@@ -100,7 +102,8 @@
            [:a.link.f7.gray.hover-dark-gray.mr2
             {:href (platf/get-field def :src-uri p)}
             (format "source (%s)" p)])
-         [:a.link.f7.gray.hover-dark-gray {:href (platf/get-field def :src-uri)} "source"]))]))
+         [:a.link.f7.gray.hover-dark-gray.mr2 {:href (platf/get-field def :src-uri)} "source"]))
+     [:a.link.f7.gray.hover-dark-gray.toggle-raw {:href "#"} "toggle raw"]]))
```

Comment:
It would be nice if the button would alternate between `raw docstring` / `formatted docstring` depending on the state of the UI.

Reply:
In some other places there has been a loose convention of prepending JS-relevant classnames with `js--` so I guess we could also make this one `js--toggle-raw`?

Reply:
done and done